src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -19,14 +19,17 @@ public class DocumentDBConfiguration : IExtensionConfigProvider
     {
         internal const string AzureWebJobsDocumentDBConnectionStringName = "AzureWebJobsDocumentDBConnectionString";
         internal readonly ConcurrentDictionary<string, IDocumentDBService> ClientCache = new ConcurrentDictionary<string, IDocumentDBService>();
-        private string _defaultConnectionString;
 
         /// <summary>
         /// Constructs a new instance.
         /// </summary>
         public DocumentDBConfiguration()
         {
             DocumentDBServiceFactory = new DefaultDocumentDBServiceFactory();
+
+            // default the connection string
+            var nameResolver = new DefaultNameResolver();
```

Comment:
This guy should probably have a static instance method so people can use DefaultNameResolver.Instance, rather than newing this up.